### ISSUE TITLE
Is there a yt tutorial for this?

### ISSUE THREAD

**Issue by TheRandomGamrTRG** [Iku/Google-Forms-to-Discord#66]
I mess things like this up a lot, and was wondering if there is a step by step tutorial for this.

**Comment by acollierr17** [comment]
There isn't, but if there's some major interest, I'll consider seriously contributing to this repository as this script has been really useful for me!

**Comment by SeedGAmingTV** [comment]
> There isn't, but if there's some major interest, I'll consider seriously contributing to this repository as this script has been really useful for me!

yes do that and if you could, add the ability to submit individual question responses because I can't figure out how to get two certain question to show up by them selves instead of the whole form
